### DISCUSSION TITLE
Logitech Speed Force Wireless works great*

### DISCUSSION THREAD

**Discussion by TylerWDoesStuff** [berarma/new-lg4ff#52] [General]
I have a Speed Force Wireless that I got from a Goodwill auction, it works great with this driver! Tested it in BeamNG.drive running (slowly and buggily) in Wine on an install of Pop!_OS, 20.04 LTS, and all aspects worked great (steering and FFB worked great, analog throttle/brake worked flawlessly), except for face buttons which seem to hit twice, rendering turn signals or gear shifters useless or hard to use. 

On another note, is it possible to get this driver working on Windows somehow? Linux is great and all but Windows is what a ton of games are built for, and Proton still isn't great as of writing.

**Comment by berarma** [comment]
If you could send an evtest output log while pressing the buttons I'd try to debug the double press issue.

Porting this driver to Windows would require a significant amount of work and knowledge of Windows internals, almost a complete rewrite. I couldn't do it for sure.

**Reply by TylerWDoesStuff** [reply]
Yeah, that's kinda the answer I was expecting, not to blame you or anything since I know next to nothing about this...

Anyways, here's the evtest log, interestingly it looks fine to me, but BeamNG running in Wine keeps detecting it as a double-press.

[evtest_log.txt](https://github.com/berarma/new-lg4ff/files/7230815/evtest_log.txt)
